.gitignore

```diff
@@ -3,3 +3,5 @@
 .project
 .settings/
 .factorypath
+
+target/
```

Comment:
adding the PR description here as comment to explain why adding this here?

Reply:
Good idea. Done.